src/braket/default_simulator/cirq/program_context.py

```diff
@@ -0,0 +1,84 @@
+from typing import List, Tuple
+
+import cirq
+import numpy as np
+from braket.ir.jaqcd.program_v1 import Results
+from cirq import Circuit
+
+from braket.default_simulator.openqasm.program_context import AbstractProgramContext
+
+# cirq.XX, cirq.YY, and cirq.ZZ gates are not the same as Braket gates
+CIRQ_GATES = {
+    "i": cirq.I,
+    "h": cirq.H,
+    "x": cirq.X,
+    "y": cirq.Y,
+    "z": cirq.Z,
+    "cnot": cirq.CNOT,
+    "cz": cirq.CZ,
+    "s": cirq.S,
+    "t": cirq.T,
+    "cphaseshift": cirq.cphase,
+    "rx": cirq.rx,
+    "ry": cirq.ry,
+    "rz": cirq.rz,
+    "swap": cirq.SWAP,
+    "iswap": cirq.ISWAP,
+    "ccnot": cirq.CCNOT,
+    "cswap": cirq.CSWAP,
+    "ccz": cirq.CCZ,
+    "ccx": cirq.CCX,
+    "measure": cirq.MeasurementGate,
+}
+
+
+class CirqProgramContext(AbstractProgramContext):
```

Comment:
last comment I think-- the file name should be updated to correspond to the class name

Reply:
It's alright. You are welcome to post as many comments in the last minute. 
I agree too.